lib/roo_on_rails/railties/database.rb

```diff
@@ -1,7 +1,7 @@
 module RooOnRails
   module Railties
     class Database < Rails::Railtie
-      if ActiveRecord::VERSION::MAJOR >= 4
+      if defined?(ActiveRecord) && ActiveRecord::VERSION::MAJOR >= 4
```

Comment:
Maybe I'm wrong but the railsy way to do this would be:

```ruby 
ActiveSupport.on_load :active_record do 
   # all the active record things
end
```